src/yggdrasil/ckr.go

```diff
@@ -55,25 +55,24 @@ func (c *cryptokey) init(core *Core) {
 // Configure the CKR routes - this must only ever be called from the router
```

Comment:
In principle, this should also get refactored away (maybe as part of `src/tuntap`, or maybe a separate library that `tuntap` uses). Not something to do now, just a reminder to look at it in the next phase of refactoring.